src/NServiceBus.AcceptanceTests/Recoverability/Retries/When_doing_flr_with_dtc_on.cs

```diff
@@ -9,7 +10,7 @@
 
     public class When_doing_flr_with_dtc_on : NServiceBusAcceptanceTest
     {
-        public const int maxretries = 5;
+        const int maxretries = 4;
             
         [Test]
         public void Should_do_X_retries_by_default_with_dtc_on()
```

Comment:
Is this test really specific to DTC? I guess the only requirement here should be that the endpoint is transactional.